File: 66-plus-one/plus-one.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        dic = []
        res = ""
        for i in range(len(digits)):
            res += str(digits[i])
        val = int(res) +1
        val_ = str(val)
        for i in range(len(val_)):
            dic.append(int(val_[i]))
        return(dic)

File: 66-plus-one/plus-one.php
class Solution {

    /**
     * @param Integer[] $digits
     * @return Integer[]
     */
    function plusOne($digits) {
        $test = (implode('' , $digits)) ;
        $resul = bcadd($test, '1');
        $result = explode(',' , $resul);
        $dic = [];
        for ($i = 0 ; $i<strlen($resul); $i++){
            array_push($dic , $resul[$i]);
        }

        return $dic;
    }
}